packages/tasks/src/hardware.ts

```diff
@@ -159,7 +159,11 @@ export const SKUS = {
 			"RTX 3060": {
 				tflops: 12.74,
 				memory: [12, 8],
-			},
+			}, 
+			"RTX 2070": {
+				tflops: 7.465,
```

Comment:
```suggestion
				tflops: 14.93,
```

Actually, we use the float16 numbers for reference :)

Reply:
> Actually, we use the float16 numbers for reference :)

Ahhhh, didn't know if it was FP16 or FP32.